### PR TITLE
[ID] Update `Twitter` 

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
Update Twitter article for Indonesian translation

### CODE REVIEW

File: wiki/Twitter/id.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
----
-outdated: true
-outdated_since: a6c6c6b27315fdf9e68f709dab2a08b085428e3b
----
-
 # Twitter
 
-## Kami juga menge-tweet!
+osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu anda dalam mendapatkan informasi di luar lingkungan osu!.
```
Comment (hikikuman) [inline wiki/Twitter/id.md:3]:
```suggestion
osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu kamu dalam mendapatkan informasi di luar lingkungan osu!.
```

File: wiki/Twitter/id.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+
+## Layanan
+
+| Avatar | Handle | Deskripsi |
+| :-: | :-: | :-- |
+| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Diperuntukkan untuk menyediakan pemberitahuan, dengan lalu lintas rendah, tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
+| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Membantu anda dengan masalah akun/komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
```
Comment (hikikuman) [inline wiki/Twitter/id.md:10]:
```suggestion
| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Membantu kamu terkait dengan masalah akun/komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
```

File: wiki/Twitter/id.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
 # Twitter
 
-## Kami juga menge-tweet!
+osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu anda dalam mendapatkan informasi di luar lingkungan osu!.
+
+## Layanan
+
+| Avatar | Handle | Deskripsi |
+| :-: | :-: | :-- |
+| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Diperuntukkan untuk menyediakan pemberitahuan, dengan lalu lintas rendah, tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
```
Comment (hikikuman) [inline wiki/Twitter/id.md:9]:
```suggestion
| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Diperuntukkan untuk menyediakan pemberitahuan, dengan lalu lintas rendah, tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
```

File: wiki/Twitter/id.md (hunk 2 above)
Comment (hikikuman) [inline wiki/Twitter/id.md:10]:
```suggestion
| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Membantu anda dengan masalah akun/komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
```

File: wiki/Twitter/id.md [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
 
-| Avatar | Handle | Dekripsi |
+| Avatar | Handle | Deskripsi |
 | :-: | :-: | :-- |
-| ![Avatar osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Sumber berita resmi dan pengumuman. Trafik rendah namun berkualitas. |
-| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Diperuntukkan untuk menyediakan pemberitahuan lalu lintas rendah tentang masalah situs dan bancho. |
-| ![Avatar Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Tidak sebatas osu! saja, tapi kami akan meninggalkan ini di sini bagi mereka yang tertarik. Lagipula, kebanyakan kehidupan ppy adalah osu!. |
-| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Membantu anda dengan masalah akun/komunitas. |
+| ![Avatar osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame "Twitter") | Sumber resmi berita dan pengumuman seputar osu!. |
 | ![Avatar BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Bantuan komedi ketika ada yang salah. |
```
Comment (hikikuman) [inline wiki/Twitter/id.md:17]:
```suggestion
| ![Avatar BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat "Twitter") | Bantuan komedi ketika ada yang salah. |
```

**Review by hikikuman — APPROVED** [review]

File: wiki/Twitter/id.md [hunk 5]
```diff
@@ -1,18 +1,24 @@
----
-outdated: true
-outdated_since: a6c6c6b27315fdf9e68f709dab2a08b085428e3b
----
-
 # Twitter
 
-## Kami juga menge-tweet!
+osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu kamu dalam mendapatkan informasi di luar lingkungan osu!.
```
Comment (mousesalt) [inline wiki/Twitter/id.md:3]:
```suggestion
osu! memiliki beberapa akun Twitter, masing-masing akun melayani untuk tujuan tertentu. Sebagian besar akun memiliki lalu lintas yang sangat rendah tetapi akan membantu kamu dalam mendapatkan informasi di luar lingkungan osu!.
```

File: wiki/Twitter/id.md [hunk 6]
```diff
@@ -1,18 +1,24 @@
...
+## Layanan
+
+| Avatar | Handle | Deskripsi |
+| :-: | :-: | :-- |
+| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Diperuntukkan untuk menyediakan pemberitahuan, dengan lalu lintas rendah, tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
+| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Membantu kamu terkait masalah akun/komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
+
+## Komunitas
+
+| Avatar | Handle | Deskripsi |
```
Comment (mousesalt) [inline wiki/Twitter/id.md:14]:
```suggestion
| Avatar | Akun | Deskripsi |
```

File: wiki/Twitter/id.md (hunk 5 above)
Comment (mousesalt) [reply wiki/Twitter/id.md:3]: this may keep better wording with more detail explanation

File: wiki/Twitter/id.md [hunk 7]
```diff
@@ -1,18 +1,24 @@
...
----
-
 # Twitter
 
-## Kami juga menge-tweet!
+osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu kamu dalam mendapatkan informasi di luar lingkungan osu!.
+
+## Layanan
+
+| Avatar | Handle | Deskripsi |
```
Comment (mousesalt) [inline wiki/Twitter/id.md:7]:
```suggestion
| Avatar | Akun | Deskripsi |
```

Comment (mousesalt) [reply wiki/Twitter/id.md:7]: don't really understand why `handle` there without translation, using kind detail help people to understand

File: wiki/Twitter/id.md [hunk 8]
```diff
@@ -1,18 +1,24 @@
...
+| :-: | :-: | :-- |
+| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Diperuntukkan untuk menyediakan pemberitahuan, dengan lalu lintas rendah, tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
```
Comment (mousesalt) [inline wiki/Twitter/id.md:9]:
```suggestion
| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Diperuntukkan untuk penyediaan notifikasi dengan lalu lintas rendah tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
```

File: wiki/Twitter/id.md [hunk 9]
```diff
@@ -1,18 +1,24 @@
...
+| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Membantu kamu terkait masalah akun/komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
```
Comment (mousesalt) [inline wiki/Twitter/id.md:10]:
```suggestion
| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Diperuntukkan untuk mengatasi masalah mengenai akun maupun komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
```

File: wiki/Twitter/id.md (hunk 8 above)
Comment (mousesalt) [reply wiki/Twitter/id.md:9]: serving and service is different

File: wiki/Twitter/id.md (hunk 9 above)
Comment (mousesalt) [reply wiki/Twitter/id.md:10]: gonna be sticking consistently with previous wording as well to helping people understand

File: wiki/Twitter/id.md [hunk 10]
```diff
@@ -1,18 +1,24 @@
...
+| :-: | :-: | :-- |
+| ![Avatar osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame "Twitter") | Sumber resmi berita dan pengumuman seputar osu!. |
+| ![Avatar BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat "Twitter") | Bantuan komedi ketika ada yang salah. |
+| ![Avatar NAT](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Berita, pengumuman, dan survey kecil-kecilan yang dikelola oleh [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (tidak dikelola oleh [osu!team](/wiki/People/The_Team#the-team)). |
 
-osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar lalu lintas sangat rendah tetapi akan membantu anda tetap mendapat informasi di luar lingkungan osu!.
+## Pribadi
 
-| Avatar | Handle | Dekripsi |
+| Avatar | Handle | Deskripsi |
```
Comment (mousesalt) [inline wiki/Twitter/id.md:22]:
```suggestion
| Avatar | Akun | Deskripsi |
```

File: wiki/Twitter/id.md [hunk 11]
```diff
@@ -1,18 +1,24 @@
...
 | :-: | :-: | :-- |
-| ![Avatar osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Sumber berita resmi dan pengumuman. Trafik rendah namun berkualitas. |
-| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Diperuntukkan untuk menyediakan pemberitahuan lalu lintas rendah tentang masalah situs dan bancho. |
-| ![Avatar Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Tidak sebatas osu! saja, tapi kami akan meninggalkan ini di sini bagi mereka yang tertarik. Lagipula, kebanyakan kehidupan ppy adalah osu!. |
-| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Membantu anda dengan masalah akun/komunitas. |
-| ![Avatar BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Bantuan komedi ketika ada yang salah. |
+| ![Avatar Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy "Twitter") | Akun pribadi [pembuat osu!](/wiki/People/peppy), yang berisi konten yang tidak selalu terkait osu!, namun masih membahas banyak hal terkait osu! juga. |
```
Comment (mousesalt) [inline wiki/Twitter/id.md:24]:
```suggestion
| ![Avatar Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy "Twitter") | Akun Twitter pribadi [pembuat osu!](/wiki/People/peppy), yang mana berisi konten tidak selalu terkait osu!, namun masih membahas banyak hal terkait osu! juga. |
```

Comment (mousesalt) [reply wiki/Twitter/id.md:24]: added detail information which account ppy have and remove double word in a sentence.

**Review by mousesalt — CHANGES_REQUESTED** [review]

**Review by mousesalt — APPROVED** [review]
:+1:

**Review by hikikuman — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**